packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -18,53 +18,26 @@
 // enumerated in the same row (or column) as the current focused view
```

Comment:
This file looks like it has a lot of changes because I moved a lot of static methods outside of `@implementation RCTFocusZone`. Didn't see a need to keep them inside the class.